plugin/trino-hive/src/test/java/io/trino/plugin/hive/benchmark/FileFormat.java

```diff
@@ -96,250 +96,254 @@
 import static org.apache.hadoop.hive.serde.serdeConstants.SERIALIZATION_LIB;
 import static org.joda.time.DateTimeZone.UTC;
 
-public enum FileFormat
+public interface FileFormat
 {
-    TRINO_RCBINARY {
-        @Override
-        public HiveStorageFormat getFormat()
-        {
-            return HiveStorageFormat.RCBINARY;
-        }
+    enum FileFormats
```

Comment:
Let's extract a top level object, e.g: `StandardFileFormats`. Let's not make it enum. We could then have something like `io.trino.sql.planner.SystemPartitioningHandle#createSystemPartitioning` which should be cleaner than mixing interface and enums. We could also hide objects like `PrestoOrcFormatWriter` that were previously hidden